src/nginx/varify/varify.go

```diff
@@ -15,6 +18,20 @@ func main() {
 	filename := os.Args[1]
 	localModulePath := os.Args[2]
 	globalModulePath := os.Args[3]
+	resolvConfPath := "/etc/resolv.conf"
+	if len(os.Args) > 4 && len(os.Args[4]) > 0 {
+		resolvConfPath = os.Args[4]
+	}
+	// https://github.com/cloudfoundry/bosh-dns-release/blob/master/jobs/bosh-dns/spec#L36-L38
+	defaultNameServer := "169.254.0.2"
```

Comment:
This logic is very CF-specific and could vary on different CF platforms. Is this default IP actually needed?